src/source/Main.scala

```diff
@@ -335,6 +338,7 @@ object Main {
       javaNonnullAnnotation,
       javaImplementAndroidOsParcelable,
       javaUseFinalForRecord,
+      kotlinRecordsSerializable,
```

Comment:
Hier wird `kotlinRecordsSerializable` verwendet, das Feld in `Spec` heisst aber `javaRecordsSerializable`?